src/Countries/Vietnam.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+use DateTimeZone;
+use IntlDateFormatter;
+use Spatie\Holidays\Calendars\ChineseCalendar;
+
+class Vietnam extends Country
+{
+    use ChineseCalendar;
+
+    public function countryCode(): string
+    {
+        return 'vn';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            // New Year's Day
+            'Tết Dương Lịch' => '01-01',
+            // Reunification Day
+            'Ngày Giải Phóng Miền Nam, Thống Nhất Đất Nước' => '04-30',
+            // Labour Day
+            'Ngày Quốc Tế Lao Động' => '05-01',
+            // Independence Day
+            'Ngày Quốc Khánh' => '09-02',
```

Comment:
For `Ngày Quốc Khánh`, Vietnam has a compensatory day off immediately before or after. I think this needs to be added.

You can see at [wikipedia](https://vi.wikipedia.org/wiki/C%C3%A1c_ng%C3%A0y_l%E1%BB%85_%E1%BB%9F_Vi%E1%BB%87t_Nam)

Reply:
Good point, fixed.